src/js/models/v2/awardsV2/BaseFinancialAssistance.js

```diff
@@ -11,6 +11,18 @@ import CorePeriodOfPerformance from './CorePeriodOfPerformance';
 import CoreExecutiveDetails from '../awardsV2/CoreExecutiveDetails';
 
 const BaseFinancialAssistance = Object.create(CoreAward);
+export const emptyCfda = {
+    total_funding_amount: 0,
+    cfda_title: '',
+    cfda_number: ''
+};
+
+const getLargestCfda = (acc, cfdaItem) => {
+    if (cfdaItem.total_funding_amount && cfdaItem.total_funding_amount > acc.total_funding_amount) {
```

Comment:
Do we know how JS will read this? because from a math stand point it reads `(cfdaItem.total_funding_amount && cfdaItem.total_funding_amount)` then `true > acc.total_funding_amount`

Reply:
This question is handled in the test case I believe.